model/src/main/java/io/spine/validation/MaxPolicy.java

```diff
@@ -46,18 +45,17 @@
 
     @Override
     @React
-    public EitherOf2<SimpleRuleAdded, Nothing> whenever(@External FieldOptionDiscovered event) {
+    public Just<SimpleRuleAdded> whenever(
+            @External @Where(field = OPTION_NAME, equals = "max") FieldOptionDiscovered event
+    ) {
         Option option = event.getOption();
-        if (!is(option, max)) {
-            return EitherOf2.withB(nothing());
-        }
         NumberRules rules = NumberRules.from(option);
         SimpleRule rule = rules.maxRule(event.getField());
-        return EitherOf2.withA(SimpleRuleAdded
-                                       .newBuilder()
-                                       .setType(event.getType())
-                                       .setRule(rule)
-                                       .build()
+        return new Just<>(SimpleRuleAdded
```

Comment:
Can we have a static factory called `just()` in the `Just` type so that we can write simply this:

```java
return just(SimpleRuleAdded.newBuilder()...);
```
?

Reply:
Will add to ProtoData in the next iteration. Here is the [issue](https://github.com/SpineEventEngine/ProtoData/issues/39).